GoogleForCreators/web-stories-wp
# Warn if running plugin from source

## Summary

Warn users if they forgot to run the build commands.

Helpful if someone just downloads the current master branch as a ZIP file and expects the plugin to "just work".

## Relevant Technical Choices

Uses an anonymous function because this cannot really be covered by unit tests and this shouldn't be disabled by others.

## To-do



## User-facing changes

<img width="1171" alt="Screenshot 2020-07-03 at 13 23 19" src="https://user-images.githubusercontent.com/841956/86464892-cc300380-bd30-11ea-9349-98392d32b055.png">

## Testing Instructions

Download the master branch and try to install the plugin on a fresh install, or simply remove the `vendor` folder to force the admin notice from appearing.

---



Fixes #1234
See #2961
